code/modules/antagonists/antag_org/antag_org_syndicate.dm

```diff
@@ -0,0 +1,100 @@
+
+/datum/antag_org/syndicate/donk //Completely random objectives, default traitor
+	name = "Donk Co."
+	intro_desc = "You are a Donk Co. agent, sent here to advance Syndicate interests. \
+		Current client is anonymous. Standard rules of engagement apply. Get the job done, and get it done right."
+
+/datum/antag_org/syndicate/hawkmoon //Theft only
+	name = "Hawkmoon Acquisitions"
+	intro_desc = "You are an incursion specialist from the Hawkmoon Acquisitions Corporation, a merchandising firm using less-than-legal methods of product procurement. \
+		Grab the goods, keep it quiet, leave no trace. We were never here."
+	objectives = list(/datum/objective/steal)
+	chaos_level = ORG_CHAOS_MILD
+
+/datum/antag_org/syndicate/arc //Only targets on-station Cargo/Service/Genetics/Virologist
+	name = "Animal Rights Consortium"
+	intro_desc = "You are a member of the Animal Rights Consortium, here to violently protest the cruel treatment of animals by megacorporations like Nanotrasen. \
+		Teach those animal abusers a lesson!"
+	objectives = list(/datum/objective/assassinateonce/animal_abuser)
+	chaos_level = ORG_CHAOS_MILD //Violent but never needs to permakill
+
+/datum/antag_org/syndicate/waffle //Assassination variants only
+	name = "Waffle Company"
+	intro_desc = "You are a contract killer under the employ of Waffle Co., a ruthless criminal entity that will go after any target, for the right price. \
+	Got a few new bounties on the docket, agent. Put 'em down however you see fit."
+	objectives = list(/datum/objective/assassinate, /datum/objective/assassinateonce, /datum/objective/maroon)
+
+/datum/antag_org/syndicate/cybersun //Mostly target Command/Security
+	name = "Cybersun Incorporated - The Inner Circle"
+	intro_desc = "You're an operative of Cybersun Incorporated's Inner Circle, an elite PMC and proxy arm of the company. \
+		Clean kills, clean thefts, clean getaway. Get it done, operative."
+	focus = 50 //Don't bully sec too hard
+	objectives = list(/datum/objective/assassinateonce/command, /datum/objective/assassinate/mindshielded, /datum/objective/steal/cybersun)
+
+/datum/objective/steal/cybersun
+	name = "Steal Item (Cybersun)"
+	steal_list = list(/datum/theft_objective/antique_laser_gun, /datum/theft_objective/nukedisc, /datum/theft_objective/hoslaser)
+
+/datum/antag_org/syndicate/interdyne //Mostly target Medical
+	name = "Interdyne Pharmaceuticals"
+	intro_desc = "You are a specialist from Interdyne Pharmaceuticals, a medical conglomerate threatened by Nanotrasen's recent forays into the medical field. \
+		Nanotrasen's medical wing has been a bit too comfortable recently. Keep 'em on their toes, specialist."
+	focus = 70
+	objectives = list(/datum/objective/assassinate/medical, /datum/objective/assassinateonce/medical, /datum/objective/steal/interdyne)
+
+/datum/objective/steal/interdyne
+	name = "Steal Item (Interdyne)"
+	steal_list = list(/datum/theft_objective/hypospray, /datum/theft_objective/defib, /datum/theft_objective/krav)
+
```

Comment:
Put these objectives in another file, not mixed in with the antag organization things